osu.Game.Modes.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -173,6 +173,7 @@ private void load(OsuColour colours)
         public override void Add(DrawableHitObject<TaikoHitObject, TaikoJudgement> h)
         {
             h.Depth = (float)h.HitObject.StartTime;
+            h.Scale = new Vector2(PLAYFIELD_SCALE);

```

Comment:
the const here mentions height scale, but this is scaling in both width and height